LICENSE

```diff
@@ -0,0 +1,24 @@
+All code submitted prior to 2020-12-22 belongs to balefull
+All code submitted after that date is released via MIT License as per below

```

Comment:
It would be nice to explicitly mention that this is a fork of Bale's work, give links for reaching Bale (if it is possible), and point to a copy of the codebase that contains his contributions only.

Some links:

- https://github.com/balefull
- https://sourceforge.net/u/balefull
- https://sourceforge.net/projects/bale-ocd/
- https://github.com/Loathing-Associates-Scripting-Society/OCD-Cleanup/tree/af3d5e5446ec37ca7e202b0b2872d3857df657d1

Reply:
Do you have a phrasing suggestion? I will alter it as suggested.

what about
This project is a fork of https://sourceforge.net/projects/bale-ocd/
All code submitted prior to 2020-12-22 is licensed as above.
All code submitted after that date is licensed under MIT License.

Reply:
That's the wrong link I found on my initial attempt. Apparently Bale has two repositories for OCD Inventory Control; one has only 1 commit while the other has +50. The latter is where we forked from. (I fixed the link above)

Could you also rephrase "licensed as above" to indicate that Bale has full copyright over his work, but has not specified a license? Something like "...is owned by Bale, under no specific license."

Reply:
change made. how does it look now?

Reply:
Good enough for me!